### ISSUE TITLE
RT import confirm and warning messages need to account for multiple RTs in a single Profile file

### ISSUE THREAD

**Issue by michelleif** [LD4P/sinopia_editor#554]
do not clear the confirm messages until the entire Profile is uploaded

what's happening now is:

Profile file contains RT1 and RT2
Person uploads file
Blue message confirms upload of RT1 and then immediately erases that and confirms upload of RT2, so it looks to the user like only RT2 was uploaded.
Before #532 was fixed, things worked as expected on a single upload (confirm would say something like "uploaded RT1 and uploaded RT2"

haven't tested the situation of uploading a Profile with 2 RTs that already exist to see whether the "RT already exists" messaging is working properly and telling you that both already exist

**Comment by michelleif** [comment]
[554.json.txt](https://github.com/LD4P/sinopia_editor/files/3214365/554.json.txt)
Profile with 2 RTs, you should get a confirm for successful upload of 
554RT1 and 554RT2

**Comment by michelleif** [comment]
Scenarios to test:
1. both templates are new == no modal + two created flash messages
2. both templates are updates == modal with both RTs referenced + two updated flash messages
3. one new, one update == modal with one RT referenced + flash message with one created and one updated

I tested scenario 2, on Safari and Firefox (Firefox was fresh browser session; Safari did hard refresh on existing). 

The modal correctly referenced both RTs, and the flash message correctly showed that it was "attempting to update" both RTs, 
<img width="1019" alt="Screen Shot 2019-05-24 at 8 41 20 AM" src="https://user-images.githubusercontent.com/23069165/58340281-4db02500-7e00-11e9-9bb5-d5421213361b.png">

but when I chose "Yes, overwrite", the final flash message referred only to one of the 2 RTs. 
<img width="998" alt="Screen Shot 2019-05-24 at 8 41 32 AM" src="https://user-images.githubusercontent.com/23069165/58340306-602a5e80-7e00-11e9-9617-da05d5bb1c20.png">
